tests/integration/cases/urllib3_expectation_dir/policy.dl

```diff
@@ -17,6 +17,9 @@ Policy("test_policy", component_id, "") :-
     build_tool_check(pip_id, "pip", "python"),
     check_facts(pip_id, _, component_id,_,_),
     check_failed(component_id, "mcn_find_artifact_pipeline_1"),
+    check_passed(component_id, "mcn_provenance_verified_1"),
+    provenance_verified_check(_, build_level, _),
```

Comment:
The `test.yaml` file for this test has not changed in this PR and `--verify-provenance` is not passed to the `analyze` command. How come this check passes? :thinking: 

I have the same comment for the other `urllib3` integration tests.

Reply:
I thought I had also disabled the level 3 verification with the command argument, but apparently not. That was overriding the base verification for these test cases.